internal/network/data_sender.go

```diff
@@ -45,25 +45,21 @@ type DataSender interface {
 	SendCanDumpData(data models.CanDumpData) error
 	// SendDeviceStatusData sends queried vehicle data over mqtt, per configuration from vehicle-signal-decoding api.
 	// The data can be gzip compressed or not
-	SendDeviceStatusData(data any) error
+	SendDeviceStatusData(data any, tokenID uint64) error
```

Comment:
Could the tokenid be at the datasender level so it doesn't need to be passed in every time?

Reply:
Yeah, I tried to avoid it since implementing the method SetTokenID is required. I thought it would be cleaner not to store tokenID in the data sender for separation of concerns. Otherwise it would be similar to what we did for vehicleInfo